pkg/gotohelm/transpiler.go

```diff
@@ -99,6 +99,7 @@ func transpile(pkg *packages.Package, deps ...string) (_ *Chart, err error) {
 			"sort.Strings":               "sortAlpha",
 			"strings.ToLower":            "lower",
 			"strings.ToUpper":            "upper",
+			"strings.HasPrefix":          "hasPrefix",
```

Comment:
Could you add a quick test for this just to showcase that the parmeters are in the right order? `lower` and `upper` only take one argument but most other string functions have the order swapped from the stdlib.

Reply:
You are right. I will add commit with tests.